src/libs/actions/Report.js

```diff
@@ -838,14 +839,15 @@ function addActions(reportID, text = '', file) {
     // Optimistically add the new actions to the store before waiting to save them to the server
     const optimisticReportActions = {};
     if (text) {
-        optimisticReportActions[reportCommentAction.clientID] = reportCommentAction;
+        optimisticReportActions[reportCommentAction.reportActionID] = reportCommentAction;
     }
     if (file) {
-        optimisticReportActions[attachmentAction.clientID] = attachmentAction;
+        optimisticReportActions[attachmentAction.reportActionID] = attachmentAction;
     }
 
     const parameters = {
         reportID,
+        reportActionID: reportCommentAction.reportActionID,
```

Comment:
I think we'll also want to add `commentReportActionID: reportCommentAction.reportActionID`

Reply:
I've added a conditional to guarantee we're using the right reportActionID in case we have attachments